src/components/HYLButton.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import '../stylesheets/HYLButton.scss';
+
+const colors = {
+    blue: "#92E0EE",
```

Comment:
If you create sass or css variables in `src/index.scss` you can use it globally in any of the styling files so will minimize the declaring of the actual hex codes